app/models/beneficiaries/IndividualBeneficiary.scala

```diff
@@ -40,32 +43,42 @@ object IndividualBeneficiary extends BeneficiaryReads {
       (__ \ 'dateOfBirth).readNullable[LocalDate] and
       __.lazyRead(readNullableAtSubPath[IndividualIdentification](__ \ 'identification)) and
       __.lazyRead(readNullableAtSubPath[Address](__ \ 'identification \ 'address)) and
-      (__ \ 'vulnerableBeneficiary).read[Boolean] and
+      (__ \ 'vulnerableBeneficiary).readNullable[Boolean] and
       (__ \ 'beneficiaryType).readNullable[RoleInCompany] and
       (__ \ 'beneficiaryShareOfIncome).readNullable[String] and
       (__ \ 'beneficiaryDiscretion).readNullable[Boolean] and
+      (__ \ 'countryOfResidence).readNullable[String] and
+      (__ \ 'nationality).readNullable[String] and
+      (__ \ 'legallyIncapable).readNullable[Boolean] and
```

Comment:
I think we can change this to `(__ \ 'mantalCapacity).readNullable[Boolean].map(_.map(!_))` then this will give you `legallyIncapable` and you won't have to call `.map(!_)` in each case

Reply:
Updated